src/main/java/uk/gov/ons/ctp/response/collection/exercise/schedule/SchedulerConfiguration.java

```diff
@@ -152,7 +154,9 @@ private static EventDTO getEventDTOFromJobKey(final Scheduler scheduler, final J
           .filter(Objects::nonNull)
           .collect(Collectors.toList());
     } catch (SchedulerException e) {
-      log.error("Failed to get job keys for group {} - {}", groupName, e.getMessage());
+      log.with("group", groupName)
+          .with("message", e.getMessage())
+          .error("Failed to get job keys for group");
```

Comment:
when we're logging an exception, we should put the exception as the final parameter in the method call to either `.warn` or `.error` and let the logging framework log the cause and the stack trace etc.